spec/order_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'order'
+
+describe Order do
+
+  let(:menu) {{:orange => 1, :noodles => 10, :pie => 6}}
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)